plugins/BLE/ble.py

```diff
@@ -1,2 +1,79 @@
-# yo deadass bluez sucks and my pi dies installing any bluetooth library
-# someone work on this pelase
\ No newline at end of file
+# Source: https://github.com/RapierXbox/ESP32-Sour-Apple
+# Licensed under GPL 3.0, we should be able to use this right? 
+# ESP32(Python) Sour Apple by @RapierXbox and @Amachik
+
+from core.plugin import BasePwnhyvePlugin
+from core.SH1106.screen import screenConsole, waitForKey
+import random
+import bluetooth._bluetooth as bluez
+from time import sleep
+import struct
+import socket
+import array
+import fcntl
+from errno import EALREADY
+
+screen = screenConsole(draw, disp, image)
```

Comment:
this would have to be moved inbetween line 31 and 32 - draw, disp and image aren't global variables (iirc)